src/main/java/lotto/money/Money.java

```diff
@@ -0,0 +1,32 @@
+package lotto.money;
+
+import lotto.system.MessageConstant;
+
+public class Money {
+    private int amount = 0;
+
+    public Money(int amount){
+        validationNegative(amount);
+        this.amount = amount;
+    }
+
+    public Money(String stringAmount){
+        try {
+            amount = Integer.parseInt(stringAmount);
+            validationNegative(amount);
+        } catch (NumberFormatException e) {
+            throw new NumberFormatException(MessageConstant.ERROR_VALID_NOT_NUMBER);
+        }
+    }
+
+    private static void validationNegative(int amount) {
```

Comment:
의미있는 객체로 사용하기 위한 검증 처리 좋습니다 👍 

참고 사항으로, 자바 컨벤션상 메서드 이름은 동사 또는 동사구로 시작하는걸 추천드립니다.

```suggestion
    private static void validateNegative(int amount) {
```